content/github/using-git/adding-a-remote.md

```diff
@@ -43,6 +43,7 @@ $ git remote add origin https://{% data variables.command_line.codeblock %}/octo
 
 To fix this, you can
 
+* Use this command instead git remote set-url origin https://{% data variables.command_line.codeblock %}/octocat/Spoon-Knife
```

Comment:
<pre>
* Use this command instead to change the existing remote, rather than setting a new one:
```sh
git remote set-url origin https://{% data variables.command_line.codeblock %}/octocat/Spoon-Knife
```
</pre>

Reply:
@ajs256 Thanks.